### ISSUE TITLE
Is right to set initialize_from_transfer FALSE?

### ISSUE THREAD

**Issue by chouscode** [EPFL-VILAB/XTConsistency#4]
Thanks first for your repository! I'm learning about your training codes and have a question about the graph param `initialize_from_transfer`. Don't we need to use the pre-training model in training steps? According to the read of paper, consistency contraints need pre-trained model for precep loss calculation. 
Hope your reply! TKS
